lnrpc/rpc.proto

```diff
@@ -647,6 +657,24 @@ service Lightning {
     };
 }
 
+message Utxo {
+
+	/// The type of address
+	string address_type = 1 [json_name = "address_type"];
+
+	/// The address
+	string address = 2 [json_name = "address"];
+
+	/// the value of the unspent coin in satoshis
+	int64 amount = 3 [json_name = "amount"];
+
+	/// the scriptpubkey in hex
+	string scriptpubkey = 4 [json_name = "scriptpubkey"];
+
+	/// The outpoint in format txid:n
+	string outpoint = 5 [json_name = "outpoint"];
+}
```

Comment:
Consider reusing the structured types `enum AddressType` and `message ChannelPoint`, to prevent string parsing client-side.

Reply:
Thanks, that's helpful. I'll take a look.

Reply:
Fixed as per 60ae2afc236c6eb3d1095cca9dfed5d9760a23b4 although I have minor questions about this, see comment below.